### DISCUSSION TITLE
Hover card language certain terms

### DISCUSSION THREAD

**Discussion by jpkoprive** [microsoft-search/pnp-modern-search#2829] [Q&A]
Hi all,

I have created an employee directory using LocalPeopleResults en set the hover card property to on. It works fine when I hover,  but the terms in the  hover card are in English language. How do I change this to Dutch? For example these terms are in English: Send email, show more, About. Skills & Experience. I have 4.8.0.0 running.

All settings in MS365 concerning my Profile are set to Dutch and the rest of the webparts I can read in Dutch.

![image](https://user-images.githubusercontent.com/48492490/223662489-44368db5-3f03-418f-b135-eb9ebe4bf0fd.png)

Kind Regards,

JP

**Comment by wobba** [comment]
See https://github.com/microsoft-search/pnp-modern-search/issues/1321 as the toolkit component does not have translation support.
